internal/provider/resource_service.go

```diff
@@ -86,6 +86,55 @@ func resourceService() *schema.Resource {
 					Type: schema.TypeString,
 				},
 			},
+			"maintainer": {
+				Description: "service owner",
+				Type:        schema.TypeList,
+				Optional:    true,
+				MaxItems:    1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"id": {
+							Description:  "The id of the maintainer.",
+							Type:         schema.TypeString,
+							Required:     true,
+							ValidateFunc: tf.ValidateObjectID,
+						},
+						"type": {
+							Description:  "The type of the maintainer. (user or team)",
+							Type:         schema.TypeString,
+							Required:     true,
+							ValidateFunc: validation.StringInSlice([]string{"user", "team"}, false),
+						},
+					},
+				},
+			},
+			"tags": {
+				Description: "service tags",
+				Type:        schema.TypeList,
+				Optional:    true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"key": {
+							Description: "key",
+							Type:        schema.TypeString,
+							Required:    true,
+						},
+						"value": {
+							Description: "value",
+							Type:        schema.TypeString,
+							Required:    true,
+						},
+					},
+				},
+			},
+			"alert_sources": {
+				Description: "List of alert source names (shortNames).",
```

Comment:
Please remove `(shortNames)`, it doesn't make sense to someone reading the docs.